### PR TITLE
Prac 09 feedback

### PR DESCRIPTION
**Pull request by RhysBush**
Sorry this is really late! Don't feel pressured to review this. @tahliaclennar23 It will be merged, but you can till comment regardless. :)

### CODE REVIEW

File: prac_09/band.py [hunk 1]
```diff
@@ -0,0 +1,28 @@
+"""
+CP1404 Band exercise
+Estimated time: 30 minutes.
+Actual time:
```
Comment (tahliaclennar23) [inline prac_09/band.py:4]: ?

File: prac_09/band.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+Rhys Bush
+"""
+
+
+class Band():
+
```
Comment (tahliaclennar23) [inline prac_09/band.py:10]: Docstring here

File: prac_09/band.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    def __init__(self, name=""):
```
Comment (tahliaclennar23) [inline prac_09/band.py:11]: Add docstrings under all your functions/methods :)

File: prac_09/unreliable_car.py [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+class UnreliableCar(Car):
+    """Represent a Car object."""
+
+    def __init__(self, name, fuel, reliability):
+        super().__init__(name, fuel)
+        self.reliability = reliability
+
+    def drive(self, distance):
+        random_number = randint(0, 101)
```
Comment (tahliaclennar23) [inline prac_09/unreliable_car.py:18]: Why not 0 to 100?

File: prac_09/taxi.py [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+        """Initialise a Taxi instance, based on parent class Car."""
+        super().__init__(name, fuel)
+        self.current_fare_distance = 0
+
+    def __str__(self):
+        """Return a string like a Car but with current fare distance."""
+        return f"{super().__str__()}, {self.current_fare_distance}km on current fare, ${self.price_per_km:.2f}/km"
+
+    def get_fare(self):
+        """Return the price for the taxi trip."""
```
Comment (tahliaclennar23) [inline prac_09/taxi.py:22]: Rounding does occur later on but not sure where that is.

File: prac_09/band.py (hunk 1 above)
Comment (RhysBush) [reply prac_09/band.py:4]: Forgot to put the actual time down.

**Pull request merged**